docs/source/data-access/data-dependency.rst

```diff
@@ -0,0 +1,87 @@
+Data Dependency Management
+==========================
+
+The IMAP science data center utilizes an event-based processing system that allows for
+processing as soon as data is available. This system is designed to be flexible to
```

Comment:
```suggestion
processing as soon as data are available. This system is designed to be flexible to
```

Reply:
Is this right? `is` sounds correct to me...

Reply:
Hmm, I thought "data" is a plural word, but I guess it is context dependent?

https://www.thesaurus.com/e/grammar/data-is-or-data-are/

Reply:
That link is interesting! The line about how computer data is usually singular is probably why I wrote it as singular. I think since this is like a general data word I'll leave it as "is" (pun not intended) but I learned something new today!